src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -169,6 +172,8 @@ private void createSymbolicLink(File srcFile, Path dst) {
     }
 
     private void downloadFile(URI uri, Path dst) {
+        urlMangling.put(goGoDuckConfig.getUrlSubstitutionKey(), goGoDuckConfig.getUrlSubstitutionValue());
+        URLMangler.setUrlManglingMap(urlMangling);
         URL url = URLMangler.mangle(uri);
```

Comment:
Perhaps this should be instantiated with the urlmap rather than calling setting static maps here